src/2.3/docker-entrypoint.sh

```diff
@@ -16,9 +16,9 @@ setting() {
 
 if [ "$1" == "neo4j" ]; then
     setting "keep_logical_logs" "${NEO4J_KEEP_LOGICAL_LOGS:-100M size}" neo4j.properties
-    setting "dbms.pagecache.memory" "${NEO4J_CACHE_MEMORY:-512M}" neo4j.properties
```

Comment:
Running with `10M` page cache is fine during cluster tests but don't change the production default